quickshear.py

```diff
@@ -31,6 +31,10 @@ def __repr__(self):
     def BibTeX(*args, **kwargs):
         pass
 
+"""Version number."""
+from importlib.metadata import version
+__version__ = version(__package__)
```

Comment:
This doesn't seem ideal, as it makes the file now need to be installed correctly with `pip`. It currently works as a standalone script.